### PR TITLE
ISSD-1341 [MARKETPLACE] Create Structure for Create License Page

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CreateLicense/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		[StepCreateLicense.SUBSCRIPTION]: {
+			backStep: StepCreateLicense.SUBSCRIPTION,
+			component: <div>Subscription Oi</div>,
+			nextStep: StepCreateLicense.LICENSE_KEY_DETAILS,
+			stepTitle: 'Subscription',
+			title: 'Subscription',
+		},
+		[StepCreateLicense.LICENSE_KEY_DETAILS]: {
+			backStep: StepCreateLicense.SUBSCRIPTION,
+			component: <div>License Oi</div>,
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CreateLicense/index.tsx:30]:
```suggestion
			component: <div>License</div>,
```

**Review by kevenleone — COMMENTED** [review]
LGTM 🔥

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CreateLicense/index.tsx (hunk 1 above)
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CreateLicense/index.tsx:30]: Done.

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:forward:force

**Pull request closed without merging**